.codeclimate.yml

```diff
@@ -22,7 +22,7 @@ engines:
     - vendor/assets/javascripts
     - client/karma.conf.js
   fixme:
-    enabled: true
+    enabled: false
```

Comment:
Why is it false now?

Reply:
Having TODO in the code is not necessarily something that should invoke a CodeClimate error whose resolution is "Must be Fixed Immediately".